k8s/core/configmap/types.go

```diff
@@ -59,7 +59,7 @@ type FatalCb func(format string, args ...interface{})
 
 type configMap struct {
 	name                   string
-	kLockV1                k8sLock
+	kLockV1                *k8sLock
```

Comment:
Why do need this to be pointer now?

Reply:
if i dont use pointer,
it travis build failed with error,
`literal copies lock value from kLockV1: github.com/portworx/sched-ops/k8s/core/configmap.k8sLock
`
it something has to do with Go's mutex type (sync.Mutex) is not safe to copy,